openwisp_users/api/serializers.py

```diff
@@ -32,6 +32,22 @@ class Meta:
             'created',
             'modified',
         )
+        extra_kwargs = {'slug': {'validators': []}}
```

Comment:
what is this doing?

Reply:
@nemesisdesign  I replied here : https://github.com/openwisp/openwisp-users/pull/334#discussion_r1061374455

Reply:
ok, can you please summarize this concept in a comment above this line?

Reply:
@nemesisdesign  Added in [`f1eb8ec` (#334)](https://github.com/openwisp/openwisp-users/pull/334/commits/f1eb8ecde45a0c7b1495019fde4c01116c42f315)